spec/07-variables.md

```diff
@@ -202,7 +202,7 @@ calls.
 
 **Constraints**
 
-Each *expression* must designate a variable name.
+Each *expression* must designate a variable name that is not the result of the non-brace form of [variable-name-creation-expression](10-expressions#variable-name-creation-operator).
```

Comment:
I made my proposed changed based on the entry titled, "global only accepts simple variables" at http://php.net/manual/en/migration70.incompatible.php. However, Neither my PHP7 not HHVM compiler seemed to know anything about it. So if that isn't going to be implemented, then feel free to ignore my pull request.